packages/frontend/src/hooks/dashboard/useDashboardRefresh.tsx

```diff
@@ -0,0 +1,23 @@
+import { useQueries, useQueryClient } from 'react-query';
+
+export const useDashboardRefresh = (dashboardUuid: string) => {
+    const queryClient = useQueryClient();
+
+    const queriesToRefresh = [
+        { queryKey: ['saved_query'] },
```

Comment:
I think we are missing `queryResults` in this list, otherwise is fetching the new chart but not new results.

Reply:
thanks! Just updated :)